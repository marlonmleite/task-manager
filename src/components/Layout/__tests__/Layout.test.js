import React from 'react'
import { shallow } from 'enzyme'
import { Col, Row } from '../'

test('should render a Col', () => {
  const col = shallow(<Col size="20px" />)

  expect(col).toMatchSnapshot()
})

test('should render a Row', () => {
  const row = shallow(<Row />)

  expect(row).toMatchSnapshot()
})
