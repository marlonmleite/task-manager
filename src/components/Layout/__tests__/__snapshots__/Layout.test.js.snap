// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Col 1`] = `
.c0 {
  box-sizing: border-box;
  -webkit-flex: 0 0 auto;
  -ms-flex: 0 0 auto;
  flex: 0 0 auto;
  max-width: 100%;
  padding: 0 15px;
  -webkit-flex-basis: 20px;
  -ms-flex-preferred-size: 20px;
  flex-basis: 20px;
  max-width: 20px;
}

<div
  className="c0"
  size="20px"
/>
`;

exports[`should render a Row 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
}

<div
  className="c0"
/>
`;
